src/appComponents/common/ApiJsonFactory.js

```diff
@@ -1,29 +1,38 @@
 /**
  * Created by championswimmer on 29/5/15.
  */
+(function () {
 
-var commonFactories = angular.module('openevent');
+"use strict";
+const commonFactories = angular.module("openevent");
+
+commonFactories.factory("ApiJsonFactory", ["$q", "$http", function ($q, $http) {
+
+const baseUrl = `(config.use_testApi ? "testapi/" : config.apiBaseGetUrl) event/ ${config.eventId}`;
 
-commonFactories.factory('ApiJsonFactory', ['$q', '$http', function ($q, $http) {
-var baseUrl = (config.use_testApi?'testapi/':config.apiBaseGetUrl) +
-    'event/' + config.eventId;
         return {
             getJson: function ($apiEndpoint) {
-                let endpoint;
-                if (($apiEndpoint === 'event') && (!config.use_testApi)) {
-                    endpoint = baseUrl;
+                let endpoint ="";
+                if (($apiEndpoint === "event") && (!config.use_testApi)) {
+                 
+                 endpoint = baseUrl;
+                
                 } else {
-                    endpoint = baseUrl + '/' + $apiEndpoint;
-                }
+                 
+                 endpoint = baseUrl + "/" + $apiEndpoint;
                 
-                var deferred = $q.defer(),
+                }
+                let deferred = $q.defer(),
                     httpPromise = $http.get(endpoint);
                 httpPromise.then(function (response) {
                     deferred.resolve(response);
                 }, function (error) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["error" is defined but never used](https://www.codacy.com/app/triplez/open-event-webapp/file/2432887269/issues/source?bid=3233857&fileBranchId=3238420#l29)